src/frontend/src/components/ChangeRequestDropdown.tsx

```diff
@@ -46,12 +47,16 @@ const ChangeRequestDropdown = ({ control, name, errors }: ChangeRequestDropdownP
 
   const approvedChangeRequestOptions = filteredRequests.map((cr) => ({
     label: `${cr.crId} - ${wbsPipe(cr.wbsNum)} - ${cr.submitter.firstName} ${cr.submitter.lastName} - ${cr.type}`,
-    value: cr.crId
+    id: cr.crId.toString()
   }));
 
-  const renderValues = new Map<number, string>();
-
-  changeRequests.forEach((cr) => renderValues.set(cr.crId, `${cr.crId} - ${wbsPipe(cr.wbsNum)}`));
+  const renderValues = new Map<number, { label: string; id: string }>();
+  changeRequests.forEach((cr) =>
+    renderValues.set(cr.crId, {
+      label: `${cr.crId} - ${wbsPipe(cr.wbsNum)} - ${cr.submitter.firstName} ${cr.submitter.lastName} - ${cr.type}`,
+      id: cr.crId.toString()
+    })
+  );
```

Comment:
You should be able to simplify this to just use approvedChangeRequestOptions now that we're using an NERAutocomplete, you can search for the approvedChangeRequestOption that has the same id as the tracked value